src/select/__tests__/select.test.js

```diff
@@ -348,7 +382,9 @@ describe('Stateless select', function() {
         .find(StyledInput)
         .first()
         .simulate('click');
-      expect(wrapper.instance().state.isDropDownOpen).toBeTruthy();
+      setTimeout(() => {
+        expect(wrapper.instance().state.isDropDownOpen).toBeTruthy();
+      }, 0);
```

Comment:
It looks like all these tests are async now (due to promise / setTimeout). Should we be using a done callback or async await to make sure the assertions are being called before test cleanup? https://jestjs.io/docs/en/asynchronous

Reply:
thanks, it works well